.ci/settings.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+printf "%s\n" "<settings>
+    <servers>
+        <server>
+            <id>sonatype-nexus-snapshots</id>
```

Comment:
Remove this server, it is not required. For release we use sonatype-nexus-stagin

Reply:
Changes Done